src/controller/CHIPDeviceController.cpp

```diff
@@ -1381,7 +1378,7 @@ void DeviceCommissioner::OnRootCertSuccessResponse(void * context)
 
     device = &commissioner->mActiveDevices[commissioner->mDeviceBeingPaired];
 
-    err = commissioner->SendOperationalCertificateSigningRequestCommand(device);
+    err = commissioner->ProcessOpCSR(device, device->GetCSR());
```

Comment:
This doesn't make sense to me. The OpCSR should be processed by the CA before adding the root cert, as your PR description implies.